Add examples for using default in terraform (#304)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/iac/terraform/terraform.md

- `value` - the value of the property (for non-array properties);
- `items` - an array of values (for array properties).

:::note

> To set a default value, use the object keyword `default_value` with the desired value as the key. For instance:

Reviewer: ```suggestion
To set a default value, use the object keyword `default_value` with the desired value as the key. For example:
```

```hcl showLineNumbers
properties {
  name = "myStringProp"
  default_value = {
    "value": "My string"
  }
}
```

> To set default values for array properties, use the default_items keyword as a desired array as its value.

Reviewer: ```suggestion
To set default values for array properties, use the `default_items` keyword with the desired array as its value. For example:
```

Reviewer: Replace all references of the terraform module version to the latest available version
